packages/maskbook/src/plugins/ITO/UI/ClaimDialog.tsx

```diff
@@ -212,13 +211,7 @@ export function ClaimDialog(props: ClaimDialogProps) {
     useEffect(() => {
         if (swapState.type === TransactionStateType.UNKNOWN) return
 
-        if (swapState.type === TransactionStateType.HASH) {
-            const { hash } = swapState
-            setTimeout(() => {
-                window.open(resolveTransactionLinkOnEtherscan(chainId, hash), '_blank', 'noopener noreferrer')
-            }, 2000)
-            return
-        }
+        if (swapState.type === TransactionStateType.HASH) return
```

Comment:
```suggestion
        // see #2796
        if (swapState.type === TransactionStateType.HASH) return
```

Reply:
typo error

Reply:
@lelenei I don't see the redirection but just deleting the original code.

In case the opening window blocked by browser, you can try:

```ts
const newWin = window.open();
newWin.location = resolveTransactionLinkOnEtherscan(chainId, hash);
newWin.focus()
```